crypto/init.c

```diff
@@ -33,6 +33,8 @@
 #include <openssl/cmp_util.h> /* for OSSL_CMP_log_close() */
 #include <openssl/trace.h>
 #include <crypto/ctype.h>
+#include <openssl/ssl.h> /* For OPENSSL_INIT_(NO_)?LOAD_SSL_STRIONGS */
```

Comment:
```suggestion
#include <openssl/ssl.h> /* For OPENSSL_INIT_(NO_)?LOAD_SSL_STRINGS */
```
I suspect this nasty reacharound is going to hurt in the long run. Wouldn't these two constants have belonged into `openssl/crypto.h` from the get-go and `openssl/ssl.h` should have pulled that header in (not only under whatever deprecation is the deprecation du jour)?

Reply:
Yes, you are right.  We could add an "unregister" function to the load-errors.  That's cleaner.  The issue is that unloading libssl leaves libcrypto with error strings pointing to unloaded space.  I'll just leave this alone for now.